src/dummy/SensorTask.cpp

```diff
@@ -0,0 +1,77 @@
+#include "SensorTask.hpp"
+#include "Arduino.h"
+#include "ArduinoJson.h"
+
+// const static char some_data[] = "This is a cool UDP Test!\n";
+// const static char some_data[] = "{tick:1,sensor:10}\n";
+
+void SensorTask::activity()
+{
+    struct netconn *conn_1;
+    int conn_check;
+
+    /* create a connection structure */
+    conn_1 = netconn_new(NETCONN_UDP);
+    /* bind the connection to port on any IP address */
+    conn_check = netconn_bind(conn_1, IP4_ADDR_ANY, 1000);
+    while (conn_check != ERR_OK)
+    {
+        LWIP_DEBUGF(LWIP_DBG_ON, ("Bind error=%d\n", conn_check));
+        netconn_delete(conn_1);
+        vTaskSuspend(this->getTaskHandle());
+    }
+
+    ip4_addr_t dst;
+    IP4_ADDR(&dst, 192, 168, 1, 1);
+    netconn_connect(conn_1, &dst, 2001);
```

Comment:
Is the netcon_bind not redundant with this netcon_connect?

Reply:
Not sure, it wasn't really working before we did this. Definitely reedit for TCP though

Reply:
Yeah it might be redundant.